src/App.tsx

```diff
@@ -1,7 +1,30 @@
 function App() {
   return (
     <>
-      <div> Hello Team !!!</div>
+      <div className="container-nav">
+        <nav>
```

Comment:
This will end up being replaced when we set up a navbar component , we can discuss why these change in a second.